pkg/build/gobuild.go

```diff
@@ -270,18 +288,39 @@ func tarKoData(importpath string) (*bytes.Buffer, error) {
 	return buf, nil
 }
 
+// getAbsolutePath returns the absolute path which is calculated by joining the current directory and
+// the relative path for this importPath to the base package.
+//
+// This is necessary because when the project is a module project since "$GOPATH/importPath" is not the
+// actual absolute path of the file.
+func (g *gobuild) getAbsolutePath(importPath string) (string, error) {
+	wd, err := os.Getwd()
```

Comment:
This seems like it would break any cases where you're building something not rooted in your current directory. For go modules, I guess that's unavoidable, but I'd like to avoid breaking anyone not using go modules yet.

Reply:
Yes! I ended up using the `go list` command to find out the absolute path, that worked just fine.